docs/flink-getting-started.md

```diff
@@ -551,6 +551,37 @@ FlinkSink.forRowData(input)
 env.execute("Test Iceberg DataStream");
 ```
 
+### Streaming upsert write
+Iceberg supports upsert based on the primary key when writing streaming data into v2 table format.
+
+You can enable `upsert`  in the following way:
+
+* Create an UPSERT Table
```

Comment:
What does this mean?

Reply:
Oh, I see. It looks like you're trying to list options. To me, this is just confusing. I'd handle it in text rather than using bullets in a list so far apart.